common/helpers/sign.c

```diff
@@ -44,7 +45,7 @@ IOTA_EXPORT char* iota_sign_address_gen(char const* const seed,
   iss_kerl_subseed(subseed, subseed, index, &kerl);
 
   iss_kerl_key(subseed, key, key_length, &kerl);
-  memset(subseed, 0, HASH_LENGTH * sizeof(trit_t));
+  memset_s(subseed,HASH_LENGTH, 0, HASH_LENGTH * sizeof(trit_t));
```

Comment:
This is no relation with this PR and no applying clang-formate.

Reply:
thanks, this is a mistake